Makefile

```diff
@@ -114,6 +113,7 @@ fetch-secrets:
 	--include "ci.id_rsa.pub" \
 	--include "opstrace-collection-cluster-authtoken-secrets.yaml" \
 	--include "dns-service-login-for-ci.json" \
+	--include "gcp-svc-acc-dev-dns-service.json" \
```

Comment:
But that part is unclear :-). Why is that needed? We don't use this Makefile in the context of the DNS service, right?